example/src/release_tests/swipeableReanimation/index.tsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import { Text, Animated as Laminated, StyleSheet } from 'react-native';
+
+import {
+  GestureHandlerRootView,
+  RectButton,
+} from 'react-native-gesture-handler';
+
+import { Swipeable } from 'react-native-gesture-handler';
+import { default as ReanimatedSwipeable } from 'react-native-gesture-handler/ReanimatedSwipeable';
```

Comment:
```suggestion
import ReanimatedSwipeable from 'react-native-gesture-handler/ReanimatedSwipeable';
```

Doesn't this work?

Reply:
In the file, it's exported as `Swipeable`, to be a drop-in replacement.
I can change it's name to `ReanimatedSwipeable` if you think that would work better.

Reply:
It's a default export, you don't have to import it under the same name it was exported (and you don't need to use `as` to rename).

What you did would be necessary if it was a named export:
```jsx
export { ReanimatedSwipeable }
```
vs
```jsx
export default ReanimatedSwipeable
```

Reply:
I see, didn't understand at first. I had no idea that's how default exports work.